UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs

```diff
@@ -71,22 +71,13 @@ public static void TransportObject(UniversalObjectPhysics objectPhysics, Vector3
 			for (var node = linkedList.First; node != null; node = node.Next)
 			{
 				var currentObj = node.Value;
-				var previous = node.Previous?.Value;
-
-				//Disconnect pulling to make it not be a problem
-				currentObj.PullSet(null, false); //TODO Test without
 
 				//Transport current
 				TransportObject(currentObj, transportTo, doTileStep, maintRoomChanceModifier);
 
-				if (previous != null && currentObj.gameObject != null)
-				{
-					//There was another object before this one, pulling it. Re-establish pulling. (But only if the current object's gameObject is not null)
-					previous.CmdPullObject(currentObj.gameObject);
-				}
-
 				//TODO: Make pulling acutally continue working across teleporters for clients, not just server
```

Comment:
when you pull someone and teleport, you stop pulling on client it seems. That's what I think he meant.